ANS_JS_template.tex

```diff
@@ -93,92 +93,92 @@
 
-\subsection{Small Modular Reactors}
-For completeness, we will address the use of small modular reactors as both tools for load following and sources of process heat. Locatelli (2015) discusses the technical and economic feasibility of load following using multiple small modular reactors (SMRs), applying the excess energy toward generating algae-biofuel and desalinated water \cite{Locatelli2015}. SMRs are generally defined as nuclear reactors under 300 MWe. They can be combined to meet the demanded energy output. SMRs are a promising possibility for NRHES due to their ability to add additional modules as electricity demand in a given region increases. Multiple SMRs have a clear means of load following, simply by shutting down those reactors whose energy output is not required due to seasonal shifts etc. This same modularity allows the energy produced from a given SMR to be diverted to an industrial process, while others are conscripted for electricity output. The modularity of the SMRs allows greater flexibility in the design of the industrial park.  Locatelli et. al. discusses the benefits as well as drawbacks of an NRHES that includes a thermal desalination process~\cite {Locatelli2015}. A desalination plant has the benefits of switching between a latent and producing state and generating a product that is readily stored. The main drawback is poor water quality and output level for when the desalination unit is restarted.
+\subsubsection{RAVEN}
+The RAVEN tool was initially built for probabilistic risk assessment. RAVEN, as a probabilistic tool, can do parametric and stochastic analyses of systems\cite{RabitiRAVEN}. For the NRHES, RAVEN is used to run different wind energy generation paths in order to generate a statistically fair representation of how renewables, in this case wind power, would likely function over any given hour. RAVEN can be used to generate the most generic seasonal wind generation over a week. A typical week can be extrapolated to represent a given season. The process can be repeated over the different seasons, which can be extrapolated to represent a year. Using synthetic generated data instead of historic data avoids the critique that the model only represents past behavior and is thus inapplicable to future trends \cite{redfoot_epiney_2016}. RAVEN can also be used to determine high-risk time series samples to ensure that high risk scenarios are accounted for in the study. RAVEN can also do probabilistic assessments such as loss of load probability and sensitivity to uncertainty analysis.
```

Comment:
Moved the discussion on RAVEN to being more tightly paired with the Modelica discussion.